### DISCUSSION TITLE
Any settings to improve JSONwriter performance

### DISCUSSION THREAD

**Discussion by markopavic** [Cinchoo/ChoETL#311] [Q&A]
Hello everybody,
I have dynamic CSV files with nested objects that I am converting to JSON.
Using ChoJSONWriter, with a CSV file of 7700 rows, 800 columns, 27MB total size, it takes around 3minutes to convert to JSON. Is there a way to improve performance?
Thank you

**Comment by Cinchoo** [comment]
sample csv with expected json output along with code helps to guide. use dotnetfiddle.

**Reply by markopavic** [reply]
Hello,
Thanks for reply. 
I'am attaching a sample CSV file and JSON generated from my sample .NET code: https://dotnetfiddle.net/uAQHk9
I need to proces dynamic CSV files with many thousands of rows and up to a thousand columns.
Also, the file may contain many nested objects/repeated fields (with unique names).
I need to integrate this processing in a web application using .NET CORE 6 as a backend and React as Frontend.

Thanks in advance for support.


[Result json demo.json](https://github.com/Cinchoo/ChoETL/files/14526267/Result.json.demo.json)
[sample_file_1_row.csv](https://github.com/Cinchoo/ChoETL/files/14526268/sample_file_1_row.csv)
